docs/jupyter.rst

```diff
@@ -0,0 +1,4 @@
+Xonsh kernel for Jupyter
+========================
+
+[placeholder]
```

Comment:
This should be filled out or removed

Reply:
Removed (for now).  I don't have a reliable install procedure to document!  I got this to work once (and ran into a runtime issue), but now I cannot get setup to recognize that it should install the jupyter kernel, either in pip or conda.  A problem for another day...